odoo/addons/base/models/ir_attachment.py

```diff
@@ -375,24 +372,18 @@ def check(self, mode, values=None):
             # when checking access rights (resource was deleted but attachment
             # was not)
             if res_model not in self.env:
-                require_employee = True
                 continue
-            elif res_model == 'res.users' and len(res_ids) == 1 and self._uid == list(res_ids)[0]:
+            if res_model == 'res.users' and len(res_ids) == 1 and self._uid == list(res_ids)[0]:
```

Comment:
Could take the opportunity to change `self._uid` to `self.env.uid`

Reply:
Alright, I change it. Thank you.